packages/backend/prisma/schema.prisma

```diff
@@ -2,27 +2,40 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider = "prisma-client-js"
+    provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "postgresql"
-  url      = env("DATABASE_URL")
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
 }
 
 model Link {
-  id    String @id @default(uuid())
-  href  String
+    id         String       @id @default(uuid())
+    href       String
+    Submission Submission[]
 }
 
 model Collection {
-  id    String @id @default(uuid())
+    id String @id @default(uuid())
 }
 
 model User {
-  id    String @id @default(uuid())
+    id String @id @default(uuid())
 }
 
 model Submission {
-  id    String @id @default(uuid())
+    id           String            @id @default(uuid())
+    link         Link              @relation(fields: [linkId], references: [id])
+    linkId       String
+    linkName     String
```

Comment:
why the linkName property is on the submission model?
Shouldn't it be in the link itself?

Reply:
good call, (linkName property was mentioned in the "model should contain:" section of the issue and the link itself was optional (included when the submission is an edit request))
but then, i guess, the submission should always contain a link, even if it is a create request

Reply:
Read the user story: If the link property is defined, the submission is change request, and the linkname + description contains the new information. Obviously, if you have better solution, you can bring that here. If no link is connected, the linkname + description will be the new link entity's fields.